[2025-02-15T00:15:24.896+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:15:20.628188+00:00 [queued]>
[2025-02-15T00:15:24.907+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:15:20.628188+00:00 [queued]>
[2025-02-15T00:15:24.908+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-02-15T00:15:24.924+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): get_location_dict> on 2025-02-15 00:15:20.628188+00:00
[2025-02-15T00:15:24.931+0000] {standard_task_runner.py:57} INFO - Started process 1091 to run task
[2025-02-15T00:15:24.936+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'get_location_dict', 'manual__2025-02-15T00:15:20.628188+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp_cqghk1l']
[2025-02-15T00:15:24.938+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask get_location_dict
[2025-02-15T00:15:25.033+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:15:20.628188+00:00 [running]> on host 2079ceed134c
[2025-02-15T00:15:25.207+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='get_location_dict' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:15:20.628188+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:15:20.628188+00:00'
[2025-02-15T00:15:25.254+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T00:15:25.282+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=get_location_dict, execution_date=20250215T001520, start_date=20250215T001524, end_date=20250215T001525
[2025-02-15T00:15:25.349+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T00:15:25.393+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
